(DOCSP-11399): Update Voice & Tone w/ JD feedback (#38)
mongodb/docs-meta: source/style-guide/style/voice-and-tone.txt

Voice and Tone
==============

The voice and tone that we use influences how people think
and feel about MongoDB.

> Voice is our style and communicates our personality to the user. Tone

Reviewer: **suggestion:** Bulleted list? Maybe italicize _Voice_ and _Tone_?

Voice and Tone Attributes
-------------------------

The following table shows attributes for voice and tone. These can be

> range in intensity. While our words should always use these basic attributes, they can also use amplified attributes when necessary.

Reviewer: **questions:** 

1. Should we define V&T "attributes"?
2. Are we clear what we mean by intensity here? Should we add a clarifying sentence? 
3. What are the differences between _basic_ and _amplified_?

**nit:** Line wrap.

Author: Based on what was already here, I sort of struggled with how to frame this. Reworked.

Consider the following best practices for voice and tone:

.. list-table::
   :widths: 50 25 25
   :header-rows: 1

>    * - Best practices

Reviewer: **question:** Why _Use_ and _Don't Use_ rather than _Use_ and _Avoid_ like on other pages? Is this a matter of strength of our intent / message around these issues?

Reviewer: **question:** Should we link to the other pages (such as second person and imperative mood -> /style-guide/writing/write-to-the-user.txt?

Author: 1. Hadn't thought about it, that language was already here. Changed to "avoid".
2. Added links.

     - The product **will prompt** you to verify the deletion.

   * - Be impactful and speak directly to a pain point.
     - Stop letting maintenance of your database take time away from

>        develping your app.

Reviewer: ```suggestion
       developing your app.
```

     - MongoDB Atlas is built for agile teams who'd rather spend time
       building apps than managing databases.

   * - It's about them, not us, and promises a solution.
     - Offload the burden of managing the cloud to a team of experts.
     - The managed cloud difference, fueled by expertise...

   * - Be simple and direct.
     - Why MongoDB Atlas? Flexible and scalable document database.   
       Available as a fully managed service.
     - Are you looking for a global cloud database service that deploys
       fully managed document databases?

   * - Write in a way that the user wants to be spoken to. Use helpful
       words and phrases that are informative, simple, clear, and easy
       to understand.

>      - You must enter the IP address of the server. Click **My
>        Server** to complete the configuration.

Reviewer: ```suggestion
     - Enter the IP address of the host. Click
       :guilabel:`My Server` to complete the configuration.
```
